game/static/game/css/board.css

```
@@ -1554,4 +1554,26 @@ body {
+/* ============================================================
+   Blindfold Mode & Flash Animation
+   ============================================================ */
+body.blindfold-mode .chessboard .piece {
+    opacity: 0 !important;
+}
+body.blindfold-mode .move-dot,
+body.blindfold-mode .capture-ring {
+    display: none !important;
+}
+
+@keyframes flashError {
```

Stylelint (17.11.1)

[error] Expected keyframe name "flashError" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)